content/_data/upgrades/2-249-1.adoc

```diff
@@ -0,0 +1,70 @@
+==== User interface improvements
+
+The Jenkins user interface includes significant user interface improvements.
+The improvements provide theming support and a more modern, attractive user interface.
+
+Some plugins may need additional changes to adapt to the latest user interface changes.
```

Comment:
It's unclear why this exists in the upgrade guide. There's nothing that admins can do about this, it's not opt-out. It's not particularly actionable.

Perhaps this last paragraph can be extended to include advice to report bugs to affected plugins? I'm currently unaware of incompatible changes except possibly related to sidepanel items, so what would plugin maintainers do (@fqueiruga?)? Do we have developer resources for adapting to changes?

Reply:
I'll delete it completely.  There were two issues that I had detected previously, but one of them was a false positive.  I was concerned that I would detect more issues during testing.  The only issue I know is related to pipeline stage view and even though it is terribly distracting, there is nothing that administrators can do to resolve it.

Reply:
Fixed in c09312545ad10c814f767ecb5536592a74963c39